testdata/script/reflect.txtar

```diff
@@ -2,7 +2,7 @@ exec garble build
 exec ./main
 cmp stdout main.stdout
 
-! binsubstr main$exe 'garble_main.go' 'test/main' 'importedpkg.' 'DownstreamObfuscated' 'SiblingObfuscated' 'IndirectObfuscated' 'IndirectNamedWithoutReflect' 'AliasIndirectNamedWithReflect' 'AliasIndirectNamedWithoutReflect' 'FmtTypeField' 'LocalObfuscated'
+! binsubstr main$exe 'garble_main.go' 'DownstreamObfuscated' 'SiblingObfuscated' 'IndirectObfuscated' 'IndirectNamedWithoutReflect' 'AliasIndirectNamedWithReflect' 'AliasIndirectNamedWithoutReflect' 'FmtTypeField' 'LocalObfuscated'
```

Comment:
I would like to see `importedpkg` replaced by `indirect` to check that only certain import paths are excluded from obfuscation. Also check that `importedpkg` can be found in the binary with `binsubstr`.

Reply:
yeah it's a fair point - I should update this PR to ensure that package import paths unrelated to reflection are excluded from the binary.

Reply:
I meant that once this is opt-in you show that one packages import path is obfuscated while another's is not although both packages use reflection.